model/src/main/proto/interaction_object.proto

```diff
@@ -57,27 +57,35 @@ message ListOfSetsOfHtmlStrings {
   StringList set_of_html_strings = 1;
 }
 
-// Structure for a ImageWithRegions object.
+// An image overlaid with labeled regions.
 message ImageWithRegions {
+  // A string that represents a filepath which will be prefixed with '[exploration_id]/assets.
   string image_path = 1;
-  repeated LabelRegions label_regions = 2;
-}
 
-message LabelRegions {
-  string label = 1;
-  Region region = 2;
-}
+  // A region of an image, including its shape and coordinates.
+  message LabelRegion {
+    // A name for a particular region.
+    string label = 1;
 
-message Region {
-  string region_type = 1;
-  NormalizedRectangle2D area = 2;
-}
+    message Region {
+      // A shape value for a particular region.
+      string region_type = 1;
```

Comment:
Could this be an enum instead of a string? Prefer using compiler-enforced types versus string values.

Reply:
currently, we only support rectangular region so using an enum wouldn't be useful

Reply:
I'm not sure I follow why an enum wouldn't be useful. As it stands, the current API is a string which can be any value, whereas an enum constrains the types to a highly limited subset.

Reply:
Okay I'll change this to a enum